### DISCUSSION TITLE
Is Redash back in active development?

### DISCUSSION THREAD

**Discussion by eliottaudry** [getredash/redash#6553] [General]
Hello!

Redash user here, I recently saw this [announcement](https://github.com/getredash/redash/discussions/5962) that the project was back in development. It was a few months ago and I see some updates on the repository, but the last release is still from 2021. I also noticed posts like [this one](https://discuss.redash.io/t/data-labels-format/9773) about bug fixes that should have been released back in the days in a version 11 that does not exists today.

I'm a bit confused about whether or not the project is still developed and we will see versions updates, does someone have recent information on that that could bring clarity?

Thanks you.

**Comment by justinclift** [comment]
Yeah, we're actively fixing and improving things, but haven't made the major focused push to get a new release tested and ready yet.  We're still doing incremental improvements and fixes a bit prior to that.

There is a `preview` image on Docker Hub that's automatically generated from all new commits to our `master` branch here.  eg when PRs are merged and so on
